packages/@okta/vuepress-site/docs/reference/releases-at-okta/index.md

```diff
@@ -82,9 +82,7 @@ Documentation for features in GA release are not marked with any icons.
 
 ## Deprecation
 
-A feature identified as Deprecated is no longer recommended and may be removed in the future.
-The recommended path forward is included in the relevant documentation, libraries, or references.
-When Okta schedules an end-of-life plan, that information is also included.
+A feature identified as deprecated is no longer recommended for use and may be removed in the future — generally this means that you shouldn't use it unless you have to. For specific cases, workarounds and recommended paths forward are included in the relevant documentation, libraries, or references. When Okta schedules an end-of-life plan, that information is also included.
```

Comment:
Maybe `end-of-service` instead of `end-of-life` plan? 
Sounds a bit like we're going to reach out and start pulling life support on people. :-D 
```suggestion
A feature identified as deprecated is no longer recommended for use and may be removed in the future — generally, this means that you shouldn't use it unless you have to. Any workarounds or recommended paths forward are included in the relevant documentation, libraries, or references. When Okta schedules an end-of-service plan, that information is also included.
```